public/js/project.js

```diff
@@ -87,7 +87,7 @@ $('#new-project').on('show.bs.modal', function() {
     webBtn.addClass('btn-primary');
   });
 
-  $("#empty-checkbox").on('change', function() {
+  $('#empty-checkbox').on('change', function() {
```

Comment:
What abou making this as another patch ?
Modification of this file doesn't seem to match with title of patch although you explained it on sub comment.

Reply:
I hope to fix it on this PR :(
I comment it on description.

Reply:
for now, I will agree to it. but from next time, let's apply one change per a patch.